Modules/AVRObj2Hex.cmake

```diff
@@ -0,0 +1,62 @@
+#=============================================================================
+# Copyright 2016 Sam Hanes
+#
+# Distributed under the OSI-approved BSD License (the "License");
+# see accompanying file COPYING.txt for details.
+#
+# This software is distributed WITHOUT ANY WARRANTY; without even the
+# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the License for more information.
+#=============================================================================
+# (To distribute this file outside of CMake-Microchip,
+#  substitute the full License text for the above reference.)
+
+
+function(avr_obj2hex target)
+    find_program(AVR_OBJ2HEX
```

Comment:
I think it would be better to name it AVR_OBJCOPY as it's more descriptive.

Reply:
Hmmm, since there is already a MicrochipBin2Hex.cmake it feels for me more suitable.
On the other hand, since I do not develop our MCU software, but manage the CI and test final hardware, this naming is a little helper, that this function gives the files for program the device ;) I know... weak arguments ^^
OBJCOPY directly fits to the called avr program, so there is a better connection for typical developer...

At the end, I found this cmake file once somewhere, it is from the owner of this Repository, and I think it would be nice, if we can discuss the naming with him (and hopefully also other users)